cms.php

```diff
@@ -1,4 +1,35 @@
 <?php
+session_start ();
+
+if (!empty($_POST ['user-name'] && $_POST ['password'])) {
+
+    $db = new PDO('mysql:host=127.0.0.1; dbname=jackdb', 'root');
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,
+        PDO::FETCH_ASSOC);
+
+    $query = $db->prepare("SELECT `password` , `user` 
+    FROM `users` WHERE `user` LIKE CONCAT('%', :user, '%');");
+
+    $query->bindParam(':user', $_POST ['user-name']);
+
+    $query->execute();
+    $out = $query->fetchAll();
+
+    $passwordOut= $out[0]['password'];
+    $userOut= $out[0]['user'];
+
+    if (password_verify($_POST ['password'], $passwordOut)) {
+        $_SESSION ['logged-in'] = 1;
```

Comment:
I think `session_start()`should go here...

Reply:
Its best practice to put them at the top of the page